client/src/app/site/common/components/start/start.component.ts

```diff
@@ -57,15 +63,8 @@ export class StartComponent extends BaseModelContextComponent implements OnInit
 
         // set the welcome title
         this.subscriptions.push(
-            this.meetingSettingsService.get(`welcome_title`).subscribe(welcomeTitle => {
-                this.startContent.welcome_title = welcomeTitle;
-            }),
-            this.meetingSettingsService.get(`welcome_text`).subscribe(welcomeText => {
-                this.startContent.welcome_text = this.translate.instant(welcomeText);
-            }),
-            this.activeMeetingService.meetingObservable.subscribe(meeting => {
-                this.meeting = meeting;
-            })
+            this.router.events.pipe(filter(event => event instanceof NavigationEnd)).subscribe(() => this.refresh()),
+            this.activeMeetingService.meetingObservable.subscribe(meeting => (this.meeting = meeting))
```

Comment:
they are probably unconcurrent, so you can use `combineLatest` to limit the triggers

Reply:
It works better without `combineLatest`.